app/main.py

```diff
@@ -1,2 +1,56 @@
+import math
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self._initialize_hash_table(8)
+        self.data = []
+        self.capacity = 0
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        self.data.append([key, value])
+        self.create_hash_table(key, value)
+        self.capacity += 1
+        load_factor = 2 / 3
+        if self.capacity >= math.floor(len(self.hash_table) * load_factor):
+            self._resize()
+
+    def __getitem__(self, item: Hashable) -> Any:
```

Comment:
The `item` argument name is ambiguous, replace it with `key`. Change the variable name `key` that you use in the `self.data` iteration to another one to preserve the incoming variable name.